We don't want meson finding the libintl library installed by Homebrew on macOS as it's
not available on all systems. The builtin setting allows Meson to find it on Linux but
on macOS it uses its builtin proxy-libintl.

--- meson.build.orig	2023-05-09 06:06:13
+++ meson.build	2023-05-09 06:02:14
@@ -2089,7 +2089,7 @@
 # FIXME: glib-gettext.m4 has much more checks to detect broken/uncompatible
 # implementations. This could be extended if issues are found in some platforms.
 libintl_deps = []
-libintl = dependency('intl', required: false, allow_fallback: false)
+libintl = dependency('intl', required: false, allow_fallback: false, method: 'builtin')
 if libintl.found()
   # libintl supports different threading APIs, which may not
   # require additional flags, but it defaults to using pthreads if
@@ -2115,7 +2115,7 @@
 if libintl.found()
   have_bind_textdomain_codeset = cc.has_function('bind_textdomain_codeset', dependencies: libintl_deps)
 else
-  libintl = dependency('intl', allow_fallback: true)
+  libintl = dependency('intl', allow_fallback: true, method: 'builtin')
   assert(libintl.type_name() == 'internal')
   libintl_deps = [libintl]
   have_bind_textdomain_codeset = true  # proxy-libintl supports it
